tasks/upload_finisher.py

```diff
@@ -486,7 +486,17 @@ def merge_report(cumulative_report, obj):
             incremental_report = obj["report"]
             parallel_idx = obj["parallel_idx"]
 
-            assert len(incremental_report.sessions) == 1
```

Comment:
from the logs, i noticed sometimes an incremental report has 0 sessions, which is weird